azavea/district-builder-dtl-pa
# Add "How did you hear about DTL?" to signup

## Overview

Add "How did you hear about DTL?" questions to signup form. 

This adds a required question ("How did you hear about DTL?") with a dropdown response. If either "Another organization" or "Other" is selected, a new required question ("Where?") shows up after.

The list of answers on the dropdown should be:
 - Teacher
 - Friend, Colleague
 - Web search
 - Social media
 - Draw the Lines event
 - Media coverage
 - Elected official
 - Another organization
 - Other

### Checklist


## Testing Instructions

 * `./scripts/update` to run migrations
 * `./scripts/server` to start server
 * Sign up and test that the "How did you hear about DTL?" appears with the above dropdown values.
 * If either "Another organization" or "Other" is selected, ensure that a text input question "Where?" appears.
 * Test that these are saved correctly.
 * Test that on login, the "My Account" popup also shows these questions similarly.

Closes [#162929838](https://www.pivotaltracker.com/story/show/162929838)
